src/launchHandler.py

```diff
@@ -101,11 +63,17 @@ def launchCallback(data):
     if fileName not in launchedFiles:
         launchMsg = launchFile(package, fileName)
     else:
-        launchMsg = killLaunchFile(package, fileName)
+        launchMsg = killLaunchFile(fileName)
     return launchMsg.message, launchMsg.isLaunched, launchMsg.fileName
 
 def listCallback(data):
-    return launchedFiles.values(), launchedFiles.keys()
+    #Get array of package names
+    packageNames = []
+    for launchedFile in launchedFiles.values():
+        
```

Comment:
Is that newline necessary ? If not, that's a nitpick from me to remove it :)

Reply:
No it's not :)